alts/src/main/java/io/grpc/alts/internal/AltsHandshakerStub.java

```diff
@@ -29,28 +29,29 @@
 /** An interface to the ALTS handshaker service. */
 class AltsHandshakerStub {
   private final StreamObserver<HandshakerResp> reader = new Reader();
-  private final StreamObserver<HandshakerReq> writer;
+  private StreamObserver<HandshakerReq> writer;
+  private final HandshakerServiceStub serviceStub;
   private final ArrayBlockingQueue<Optional<HandshakerResp>> responseQueue =
       new ArrayBlockingQueue<>(1);
   private final AtomicReference<String> exceptionMessage = new AtomicReference<>();
 
   private static final long HANDSHAKE_RPC_DEADLINE_SECS = 20;
 
   AltsHandshakerStub(HandshakerServiceStub serviceStub) {
-    this.writer =
-        serviceStub
-            .withDeadlineAfter(HANDSHAKE_RPC_DEADLINE_SECS, SECONDS)
-            .doHandshake(this.reader);
+    writer = null;
```

Comment:
Delete this line. It does nothing. Initialization is only required for `final` fields. Other fields default to 0/null. Ditto below.